ripple/src/main/webapp/style.css

```diff
@@ -0,0 +1,224 @@
+
+:root {
+  /* Color variables */
+  --default-hover-seagreen-color: #87C4AD;
```

Comment:
Not really specific to this PR (and should not block it), but I noticed that a lot of these have similar values like `#707070`. Does it make sense to have like a primaryColor (or secondaryColor) defined? That might be a good question to ask UX and how they approach this. You want to make sure everyone is using the same vocab/names.

Reply:
Waiting for response from UX. Will otherwise use primary / secondary naming practice.